lightkurve/lightcurve.py

```diff
@@ -1703,6 +1708,120 @@ def errorbar(self, **kwargs):
             ax.set_xlabel("Phase")
         return ax
 
+    def waterfall(self, ax=None, bin_points=1, phmin=-0.5, phmax=0.5, bin_method='mean',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [FoldedLightCurve.waterfall is too complex (18)](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4938792)